server/pkg/analytics/analytics.go

```diff
@@ -44,26 +47,71 @@ func Handler() {
 	}
 }
 
-// UpdateAnalyticsData sends the GET request to the GA instance and receives the events' metrics at every t time intervals
-// and updates the global JSON object for containing the response
-func UpdateAnalyticsData() error {
-	GAResponseJSONObject = nil
-	key, err := ioutil.ReadFile("/etc/analytics/auth.json")
+func responseCheck(object GAResponseJSON, label string) error {
```

Comment:
instead of defining the `chaosOperatorCount` outside the function you can return the value from this function itself